rope/contrib/findit.py

```diff
@@ -118,7 +118,7 @@ def __init__(self, occurrence):
         self.lineno = occurrence.lineno
 
     def __repr__(self):
-        pymod = self.resource.project.get_pymodule(self.resource)
+        self.resource.project.get_pymodule(self.resource)
```

Comment:
```suggestion
```

This line should just be removed completely, as it doesn't really do anything; I think it was just a leftover from experimenting with different formats for the `__repr__`.